src/components/ConversationSettings/LobbySettings.vue

```diff
@@ -140,6 +148,33 @@ export default {
 				valueType: 'timestamp',
 			}
 		},
+
+		showRelativeTime() {
+			return this.lobbyTimer
+				&& this.lobbyTimer > Date.now()
+				&& (this.lobbyTimer - Date.now()) < ONE_DAY_IN_MS // less than 24 hours
+		},
+
+		getTimeZone() {
+			if (!this.lobbyTimer) {
+				return ''
+			}
+			const date = new Date(this.lobbyTimer)
+			return t('spreed', `Start time: ${date}`)
+		},
+
+		getRelativeTime() {
+			const diff = this.lobbyTimer - Date.now()
+			const hours = Math.ceil(diff / (60 * 60 * 1000))
+			if (hours > 1) {
+				// TRANSLATORS: hint for the time when the meeting starts
+				return n('spreed', 'in less than %n hour', 'in less than %n hours', hours)
```

Comment:
Maybe use Math.floor for consistency? What's the usual pattern for these cases?

```suggestion
				return n('spreed', 'in %n hour', 'in %n hours', hours)
```

Reply:
I avoided floor because it is misleading IMO: 

left 1h59min -> shows in 1 hour.

Reply:
Same would be in opposite to 1h02min. But anyway, as long as it's consistent in both places (here and at LobbyScreen.vue), I don't mind

Reply:
We should be exact to minutes here. Can't we say "In 1 hour 24 minutes"?

Reply:
Short form "In 1h 24m" also works if space is low.